.drone.star

```diff
@@ -2,27 +2,27 @@ def main(ctx):
   versions = [
 
     {
-      'value': '10.6.0',
-      'qa': 'https://download.owncloud.org/community/testing/owncloud-complete-20201216-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.org/community/owncloud-complete-20201216.tar.bz2',
-      'tarball_sha': '3853bb528c1d4d274837faced9b3e27d343e9eadbb5cf692e5eeacb91083e529',
+      'value': '10.7.0beta2',
```

Comment:
Use `10.7.0-beta2`

Reply:
thanks.
path fixed
hyphen added. (how do you remember, when a version number has to be without and when with a dash?)

Reply:
> hyphen added. (how do you remember, when a version number has to be without and when with a dash?)
Dockerism: use a hyphen
oc10 -ism: don't use hyphens (historic reasons)